app/views/teams/index.html.erb

```diff
@@ -1,21 +1,37 @@
 <% content_for :title_bar do %>
   <h1 class="top-bar-title"><%= t('teams.switch') %></h1>
 <% end %>
-<div class="row">
-  <div class="col-xs-12 col-sm-6">
-    <div class="form-wrapper">
-      <%= form_tag(switch_teams_path, method: :get) do %>
-        <div class="field-wrapper">
-          <%= label_tag :team_slug, 'Teams', class: 'col-sm-2 control-label' %>
-          <div class="col-sm-10">
-            <%= select_tag :team_slug, options_from_collection_for_select(@teams, :slug, :name), disabled: @teams.blank? %>
+
+<div class="col-md-8 col-md-offset-2">
+  <div class="page-header">
+    <h4 class="page-header-title darkgrey-2">
+      <i class="mi md-20">dashboard</i> <%= t('teams.select') %>
+      <%= link_to t('teams.create'), new_team_path, class: "btn btn-link pull-right unstyled-link" %>
+    </h4>
+  </div>
+
+  <% @teams.each_with_index do |team, index| %>
+    <%= link_to switch_teams_path(team_slug: team.slug), class: 'card-link' do %>
+      <div class="col-md-4 col-xs-6">
+        <div class="panel panel-default card clickable scaleIn-animation" style="animation-delay: <%= index.to_f * 0.15 %>s" data-form-submit>
```

Comment:
quick note:
you can replace `index.to_f * 0.15` with just `index* 0.15` and the result will be a float